distributed/bokeh/worker_monitor.py

```diff
@@ -60,6 +68,13 @@ def resource_append(lists, msg):
         lists[k].append(mean(pluck(k, L)) / 100)
 
     lists['time'].append(mean(pluck('time', L)) * 1000)
+    net = mean(pluck('network-send', L, 0))
+    if len(lists['time']) >= 2:
+        t1, t2 = tail(2, lists['time'])
```

Comment:
Yep, already ahead of you; I thought we were fixing the queues to be not-too-long, but this is fine now.